Assets/LeapMotionModules/UI/Interaction/InteractionManager.cs

```diff
@@ -0,0 +1,293 @@
+﻿using Leap.Unity.Attributes;
+using Leap.Unity.RuntimeGizmos;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Leap.Unity.UI.Interaction {
+
+  public partial class InteractionManager : MonoBehaviour, IRuntimeGizmoComponent {
+
+    [Header("Interactions")]
+    public bool enableHovering = true;
```

Comment:
Eventually going to need to hide this behind properties so that you can take proper action if they are messed with at runtime.